src/App.js

```diff
@@ -6,27 +6,49 @@ import Rank from './components/Rank';
 import Newsletter from './components/Newsletter'
 import Data from './components/Data'
 import Footer from './components/Footer';
+import chartData from './data.json';
 
 class App extends Component {
 
     constructor() {
         super();
         this.state = {
-            cDate: {}
+            cData: {},
+            currentLang: chartData[0][0].name,
```

Comment:
Let's create an object here encompassing all of these items instead of individual items. Unless you see a good reason to keep these separate? It would just be easier this way to pass data down the components